2019Labs/CustomSecurityContent/setup/system_setup.yml

```diff
@@ -6,11 +6,12 @@
   tasks:
   - name: Enable Ansible repository
     yum_repository:
-      name: ansible-2.8-for-rhel-8-x86_64-rpms
+      name: ansible-2.9-for-rhel-8-x86_64-rpms
```

Comment:
you changed here to 2.9 but the description and baseurl still uses 2.8. Is it correct?

Reply:
Good catch, fixed!